README.yaml

```diff
@@ -70,13 +74,13 @@ introduction: |-
   ### cloudposse/packages
   
   On Debian, run:
- 
+  
   ```
   apt install atmos@="<ATMOS_VERSION>-*"
   ```
   
   On Alpine, run:
- 
+  
```

Comment:
~~~suggestion
  Install the Cloud Posse package repository hosted by cloud smith.

```
curl -sSL https://apk.cloudposse.com/install.sh | bash
```
~~~